terraform/terraform_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+require 'docker'
+require 'serverspec'
+
+describe "Terraform image" do
+    before(:all) {
+        set :docker_image, find_image_id('terraform:latest')
+    }
+
+    it "installs the Alpine linux" do
+        expect(file("/etc/alpine-release")).to be_file
+    end
+
+    it "installs Root Certificates" do
+        expect(file("/usr/share/ca-certificates/mozilla/GlobalSign_Root_CA.crt")).to be_file
+    end
+
+    it "checks if terraform binary is executable" do
+        expect(file("/usr/local/bin/terraform")).to be_mode 755
+    end
+
+    it "installs all the whitelisted binaries" do
+        command("echo $BINARY_WHITELIST").stdout.split.each { |f|
+            expect(file("/usr/local/bin/#{f}")).to be_mode 755
+        }
```

Comment:
Well, one of the issues I found is that if you do a typo in the Whitelist, it will run clean and don't tell you that it missed one.

But OK, I will add an additional test and also change the title of this one.